RELEASE_NOTES.md

```diff
@@ -1,60 +1,19 @@
-# Cosmos SDK v0.43.0 Release Notes
+# Cosmos SDK v0.43.0-RC3 Release Notes
 
-This release introduces several new important updates to the Cosmos SDK. The release notes below provide an overview of the larger high-level changes introduced in the v0.43 release series.
+This is a fourth Release Candidate for v0.43. If no issue will be reported this will be the final release.
 
-That being said, this release does contain many more minor and module-level changes besides those mentioned below. For a comprehsive list of all breaking changes and improvements since the v0.42 "Stargate" release series, please see the [CHANGELOG](https://github.com/cosmos/cosmos-sdk/blob/release/v0.43.x/CHANGELOG.md).
+This release contains two major bug fixes:
 
-## Two new modules: `x/authz` and `x/feegrant`
+- [\#9841](https://github.com/cosmos/cosmos-sdk/pull/9841) Fix the bug where it was impossible to create IBC channels after an on-chain upgrade to 0.43. Please note that this is an **API-Breaking change**: `x/capability`'s `InitializeAndSeal()` has been removed and replaced with `Seal()`.
```

Comment:
Let's also mention that:
+ x/capability BeginBlock must be added to app module manager begin blockers and it must be before any other module which uses x/capability.

Reply:
I you copy or use something from: https://github.com/cosmos/cosmos-sdk/pull/9852/files#diff-68d24fa81558aae3d8c59e2aa57a4fa719ea3b04d7fa14beff45f16f00858f50R3

Reply:
Done, thanks